src/projects.md

```diff
@@ -1,23 +1,102 @@
 ---
 title: Our Projects
-date: 2015-11-03T02:55:22-07:00
-author: elina
+date: 2020-10-16T01:15:35+00:00
+author: Jess Sand
 layout: page
 ---
 
-In response to member feedback, OpenOakland is reevaluating how our projects are vetted and adopted, with a focus on ensuring all projects are aligned with [OpenOakland’s values](https://openoakland.org/our-values) and are truly serving the communities they impact. The projects listed here are currently active but will be going through this new process as it’s developed. If you’d like to contribute to or provide feedback on any of these projects or the evaluation process itself, consider attending our [next Hack Night](https://www.meetup.com/OpenOakland/events/), joining the conversation in our [Slack workspace](http://slack.openoakland.org/), or emailing our [Steering Committee](mailto:steering@openoakland.org).
+ - [Active projects](#ongoing-projects-that-support-openoakland-s-operation)
+ - [Inactive projects](#inactive-projects)
+ - [Decommissioned projects](#decommissioned-projects)
+ - [Becoming an OpenOakland project](#becoming-an-openoakland-project)
+ - [Providing feedback](#providing-feedback)
 
 ---
 
-<div id="projects">
-  <h2>Active Projects</h2>
-  {% for project in site.data.active_projects %}
-    {% include project.html %}
-  {% endfor %}
-  <hr/>
-  <h2>Inactive Projects</h2>
-  <p><em>These projects are some of OpenOakland’s original projects. They are no longer actively being developed and aren’t accepting volunteers.</em></p>
-  {% for project in site.data.inactive_projects %}
-    {% include project.html %}
-  {% endfor %}
+
+{% include linkedHeading.html heading="Ongoing projects that support OpenOakland's operation" level=2 %}
+
+These projects are a great way to get your feet wet as you get to know OpenOakland.
+
+<div id="projects-openoakland">
+{% for project in site.data.openoakland_projects %}
+{% include project.html %}
+{% endfor %}   
 </div>
+
+---
+
+{% include linkedHeading.html heading="Active projects" level=2 %}
+
+The following projects are official OpenOakland projects that are being actively supported by an existing project team. If you see something that interests you, there are a couple of ways you can get involved:
```

Comment:
The following ~~projects~~ are official OpenOakland projects that are ~~being~~ actively supported ...